hack/templates/resources/90-shoot.yaml.tpl

```diff
@@ -25,22 +25,22 @@
   kubernetesVersion=""
   if cloud == "aws":
     region="eu-west-1"
-    kubernetesVersion="1.13.2"
+    kubernetesVersion="1.13.3"
   elif cloud == "azure" or cloud == "az":
     region="westeurope"
-    kubernetesVersion="1.13.2"
+    kubernetesVersion="1.13.3"
   elif cloud == "gcp":
     region="europe-west1"
-    kubernetesVersion="1.13.2"
+    kubernetesVersion="1.13.3"
   elif cloud == "alicloud":
     region="cn-beijing"
-    kubernetesVersion="1.13.2"
+    kubernetesVersion="1.13.3"
```

Comment:
Change to 1.11.7, Alicloud does not support 1.13.

Reply:
Thank you for the spot.